README.md

```diff
@@ -0,0 +1,58 @@
+# PLD Generation Tool
+
+The PLD Generation Tool is tool that we made to help us generate good looking pdf
```

Comment:
The PLD Generation Tool is a tool that we made to help us generate good looking pdf files